### PR TITLE
add test

### PR DESCRIPTION
**Pull request by zgpeace**
add test

### CODE REVIEW

File: example_chat_completion.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates.
+# test
```
Comment (zgpeace) [inline example_chat_completion.py:2]: what's your purpose?

**Review by rexionmars — COMMENTED** [review]
what's your purpose?

**Comment by jspisak** [comment]
+1 - i am unclear the goal of this PR. Can you provide a description?

**Comment by jspisak** [comment]
Closing this PR for now as it is unclear what the purpose is. Feel free to reopen if I've made a mistake..

**Pull request closed without merging**